modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/ObjectEntryManagerImpl.java

```diff
@@ -364,6 +374,45 @@ private String _getObjectEntryPermissionName(long objectDefinitionId) {
 		return ObjectDefinition.class.getName() + "#" + objectDefinitionId;
 	}
 
+	private void _hasPreventDeletionType(long objectDefinitionId) {
+		List<ObjectRelationship> objectRelationships =
```

Comment:
since we only use `objectRelationships` once in the FOR, could you replace the right site of the FOR instead of creating the variable? Please, check the SF if it will allow :+1:

Reply:
It`s all right about SF with this way